recipes/pedcheck/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pedcheck" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/haraldgrove/{{ name }}/archive/master.zip
+  sha256: 239a9c31793cab0f4e4374ba01d174c018f836b41867870d493663c16f78fcc3
+  
+build:
+  noarch: python
```

Comment:
``noarch`` recipes does not rely on platforms. However, in your ``build`` requirements you need the ``c`` compiler, as the compilation is platform dependent this recipe can be ``noarch`` or this project needs to be compiled?